src/color/Color.ts

```diff
@@ -254,12 +255,14 @@ export class Color {
    * @see http://http://www.w3.org/TR/css3-color/#hsl-color
    */
   static sourceFromHsl(color: string): TRGBAColorSource | undefined {
-    const match = color.match(reHSLa());
+    const noNone = color.replaceAll('none', '0');
```

Comment:
what is this none use case? is it specific of svg? in that case it should be solved in the svg parser ( in a different PR )

Reply:
yes svg allows you to use 'none' instead of '0'. I will move it to the parser

Reply:
no leave it for another PR. don't keep adding stuff to the same pr. That is a different topic, 'none' is probably already handled somewhere